.github/workflows/integration_test.yaml

```diff
@@ -19,66 +18,24 @@ jobs:
             value: "--num-units=3"
           - name: num_units=1, db_from_relation
             value: "--num-units=1"
-        exclude:
-          - module: test_error
-            args:
-              name: num_units=3, db_from_config
-              value: "--test-db-from-config --num-units=3"
-          - module: test_error
-            args:
-              name: num_units=1, db_from_config
-              value: "--test-db-from-config --num-units=1"
-          - module: test_upgrade
-            args:
-              name: num_units=3, db_from_relation
-              value: "--num-units=3"
-          - module: test_upgrade
-            args:
-              name: num_units=1, db_from_relation
-              value: "--num-units=1"
-
-    name: Integration Tests (${{ matrix.module }}, ${{ matrix.args.name }})
-
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    with:
+      setup-devstack-swift: true
+      pre-run-script: |
+        -c "sudo microk8s config > kube-config && \
+        sudo microk8s enable registry hostpath-storage ingress && \
```

Comment:
ingress should be already enabled https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml#L245
Why do you need the registry?

Reply:
The registry was needed for accessing built docker image, which is totally unnecessary since the operator-workflows does it for us already!
Thank you for the comment :) I'll be updating the tests accordingly